tutorials/Part 0 - (Getting Started) Local Tokenization.ipynb

```diff
@@ -471,7 +508,9 @@
      "text": [
       "         I |  True | 5943131912006430202\n",
       "        am |  True | 11728213064939857863\n",
-      "tokenizing |  True | 5460367124718174413\n",
+      "     token | False | 12977505502605755571\n",
+      "         # | False | 15748096671724166044\n",
+      "     izing |  True | 7913766558530700748\n",
```

Comment:
if you tokenize the same string in spaCy, it will give

```
I
am
token#izing
a
PySyft
String
object
```

Can you explain this behaviour?

Reply:
Oops!, actually I did not executed this cell, I only executed initial cells.(The output of this cell is from my old closed PR for tokenizer)
SyferText Tokenizer also tokenizes the same way as Spacy!

Reply:
Also I have already added a test for such cases
`# check correct infix are tokenized only
    str1 = "Hell#o"  # this string does not contains infix.
    str2 = "Hell-o"  # This string contains logical infix.

    assert len(nlp(str1)) == 1  # ['Hell#o']
    assert len(nlp(str2)) == 3  # ['Hell', '-', 'o']`